src/schema/users.ts

```diff
@@ -1104,16 +1117,14 @@ export const resolvers: IResolvers<any, Context> = {
     },
     personalizedDigest: async (
       _,
-      {
-        type = UserPersonalizedDigestType.Digest,
-      }: { type?: UserPersonalizedDigestType },
+      __,
       ctx: Context,
-    ): Promise<GQLPersonalizedDigest> => {
+    ): Promise<GQLPersonalizedDigest[]> => {
       const personalizedDigest = await ctx
         .getRepository(UserPersonalizedDigest)
-        .findOneBy({ userId: ctx.userId, type });
+        .findBy({ userId: ctx.userId });
 
-      if (!personalizedDigest) {
+      if (!personalizedDigest || personalizedDigest.length === 0) {
```

Comment:
its always gonna be array so latter is enough

Reply:
```suggestion
      if (personalizedDigest?.length === 0) {
```
maybe?

Reply:
it will never be undefined, `findBy` returns array